lib/gc/ExecutionEngine/GPURuntime/ocl/GpuOclRuntime.cpp

```diff
@@ -20,6 +20,33 @@
 #include "mlir/Dialect/LLVMIR/LLVMDialect.h"
 #include "mlir/Pass/PassManager.h"
 
+#ifdef GC_ENABLE_GPU_PROFILE
+
+void EnableProfiling();
+void DisableProfiling();
+
+class tracer_t {
```

Comment:
Could you please follow the same naming style as in the other classes?

Reply:
Thanks for the review! The name of this class has been changed